### PR TITLE
Add tracing-web to the list of related crates

### LINKED ISSUE #2280: Introducing tracing-web to the ecosystem
The [tracing-wasm](https://github.com/storyai/tracing-wasm) crate - which is currently linked in the Related Crates section - seems to have several open issue that might take a lot of effort and re-engineering to fix. Most prominently, it doesn't offer a configurable layer based subscriber (the layer handles formatting, events and spans all in one).

I've written [tracing-web](https://crates.io/crates/tracing-web) to fix several of these issues and implemented a `tracing-subscriber` based, web browser/wasm compatible tracing subscriber. Could you please add this to the list?

If you have a lot of time, you are also invited to try it out and give me a bit of feedback :)

If you want me to write the PR to add myself to the list, I can do that too.

### PR DESCRIPTION
**Pull request by WorldSEnder**
Fixes #2280 

## Motivation

I've implemented an alternative to [`tracing-wasm`](https://github.com/storyai/tracing-wasm) to fix some issues that I had when using it (most are open as longer time bug reports, no need to recount them here). It should be more up to date with the current tracing framework and, being layer based, is easier to compose with other subscribers, e.g. supports formatting and timing on event logs out of the box.

## Solution

Add the [`tracing-web`](https://crates.io/crates/tracing-web/) crate to the list of related crates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -392,6 +392,8 @@ are not maintained by the `tokio` project. These include:
   _inside_ of functions.
 - [`tracing-wasm`] provides a `Collector`/`Subscriber` implementation that reports
   events and spans via browser `console.log` and [User Timing API (`window.performance`)].
+- [`tracing-web`] provides a layer implementation of level-aware logging of events to web browsers' `console.*` and span events to
+  the [User Timing API (`window.performance`)].
```
Comment (hawkw) [inline README.md:396]: let's wrap this line at the same column as the other lines in the file?

**Review by hawkw — COMMENTED** [review]
thanks for adding your crate, this is great!

there's also a similar list of related crates in `tracing`'s `lib.rs` documentation. can you add a link to your crate here as well? https://github.com/tokio-rs/tracing/blob/41337ba9965295957e5d6fc7e575bbd79840463f/tracing/src/lib.rs#L770-L807

thanks!

File: README.md (hunk 1 above)
Comment (WorldSEnder) [reply README.md]: I hope I got the line correctly, I didn't find any guidance on the exact number. Is there a rustfmt.toml file somewhere that specifies this?

**Review by hawkw — APPROVED** [review]
thanks, looks good to me! :)

**Pull request merged**